assets/css/less/admin.less

```diff
@@ -9,10 +9,12 @@
 	--orange: #F15A24;
 }
 
+@import 'colors.less';
```

Comment:
I don't think there will be enough in this file to make it worth having. Can the colors go in the existing common.less file instead?